installing/installing_bare_metal/bare-metal-postinstallation-configuration.adoc

```diff
@@ -17,6 +17,11 @@ include::modules/nw-enabling-a-provisioning-network-after-installation.adoc[leve
 // Creating a manifest object that includes a customized `br-ex` bridge
 include::modules/creating-manifest-file-customized-br-ex-bridge.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network]
```

Comment:
* xref:../../../installing/installing_bare_metal/bare-metal-expanding-the-cluster.adoc#installing-bare-metal.adoc_installing-bare-metal[Expanding the cluster]